sda-commons-client-jersey/src/test/java/org/sdase/commons/client/jersey/oidc/OidcRequestFilterTest.java

```diff
@@ -147,9 +147,10 @@ private void setupTokenEndpoint(boolean withBody) {
 
     if (withBody) {
       response.withBody(
-          "{\"access_token\": \"xxxx.yyyy.zzzz\",\n"
-              + "  \"expires_in\": 60,\n"
-              + "  \"token_type\": \"Bearer\"}");
+          """
+            {"access_token": "xxxx.yyyy.zzzz",
+              "expires_in": 60,
+              "token_type": "Bearer"}""");
```

Comment:
NIT:

wouldn't it be nicer to format this more readable?

```suggestion
          """
            {
              "access_token": "xxxx.yyyy.zzzz",
              "expires_in": 60,
              "token_type": "Bearer"
            }
          """);
```

Reply:
Same for a few other multiline Jsons.

Reply:
I mostly used IntelliJ's automatic actions to convert code. But you're right, I can do it better manually. I will go through the multiline changes.